File: tests/apitests/apilib/project.go
package apilib

import ()

//Project for go test
type Project struct {
	ProjectID    int32  `json:"id,omitempty"`
	OwnerID      int32  `json:"owner_id,omitempty"`
	ProjectName  string `json:"project_name,omitempty"`
	CreationTime string `json:"creation_time,omitempty"`
	Deleted      int32  `json:"deleted,omitempty"`
	UserID       int32  `json:"user_id,omitempty"`
	OwnerName    string `json:"owner_name,omitempty"`
	Public       bool   `json:"public,omitempty"`
	Togglable    bool   `json:"togglable,omitempty"`
}

File: src/clarity-seed/src/app/project/project.ts
/*
  [
    {
        "project_id": 1,
        "owner_id": 1,
        "name": "library",
        "creation_time": "2017-02-10T07:57:56Z",
        "creation_time_str": "",
        "deleted": 0,
        "owner_name": "",
        "public": 1,
        "Togglable": true,
        "update_time": "2017-02-10T07:57:56Z",
        "current_user_role_id": 1,
        "repo_count": 0
    }
  ]
*/
export class Project { 
    project_id: number;
    owner_id: number;
    name: string;
    creation_time: Date;
    creation_time_str: string;
    deleted: number;
    owner_name: string;
    public: number;
    Togglable: boolean;
    update_time: Date;
    current_user_role_id: number;
    repo_count: number;
}